configLoader.js

```diff
@@ -0,0 +1,438 @@
+const defaultBlockedFields = new Set(["tlcMapUniqueId"]);
```

Comment:
@MufengNiu , both config-loader.js and collection-config-loader.js have the list of map styles defined. Can you put it into a separate function for checking map styles instead of duplicating them?